### PR TITLE
Document new current_* sensors for xiaomi_miio vacuum.

### PR DESCRIPTION
**Pull request by OGKevin**
## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [x] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/61098
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [ ] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by MartinHjelmare** [comment]
This should target `next` branch I think.

**Review by rytilahti — APPROVED** [review]
LGTM, the parent has already been merged too.

edit: looks like that linters are not being run for some reason, and their result is required to be able to merge this.. :-)

**Comment by OGKevin** [comment]
Who has the power to kick the pipline?

File: source/_integrations/xiaomi_miio.markdown [hunk 1]
```diff
@@ -1402,13 +1402,17 @@ Main Brush Left*:
...
+Current Clean Time:
+  description: The current cleaning time of the vacuum. If the vaccuum is not cleaning, this sensor will have the same value as the "Last Clean Duration" sensor.
+Current Clean Area:
+  description: The current area that has been cleaned. If the vacuum is not cealning, this sensor will have the same value as the "Last Clean Area" sensor.
 Last Clean Area*:
   description: The last cleaned area in square meters
 Last Clean Duration*:
   description: The last clean duration in seconds
-Last Clean End*:
+Last Clean End:
```
Comment (rytilahti) [inline source/_integrations/xiaomi_miio.markdown:1413]: Just one more thing, were these sensors previously marked incorrectly as not being enabled per default?

edit: note for the future, looks like reopening the PR triggers the CI, thanks @balloob!

Comment (OGKevin) [reply source/_integrations/xiaomi_miio.markdown:1413]: correct

File: source/_integrations/xiaomi_miio.markdown [hunk 2]
```diff
@@ -1402,13 +1402,17 @@ Main Brush Left*:
   description: How long the main brush can be used in seconds
 Sensor Dirty Left*:
   description: How long the sensor can  be used in seconds
+Current Clean Time:
+  description: The current cleaning time of the vacuum. If the vaccuum is not cleaning, this sensor will have the same value as the "Last Clean Duration" sensor.
+Current Clean Area:
+  description: The current area that has been cleaned. If the vacuum is not cealning, this sensor will have the same value as the "Last Clean Area" sensor.
```
Comment (MartinHjelmare) [inline source/_integrations/xiaomi_miio.markdown:1408]: cleaning

Comment (rytilahti) [reply source/_integrations/xiaomi_miio.markdown:1408]: Mmh, true. I wish there'd be a linter for such.. :-)

**Pull request merged**